ai_flow/test/cli/commands/ut_workflows/workflows/test_command/test_command.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import unittest
+
+from ai_flow.ai_graph.ai_graph import current_graph
+from ai_flow.ai_graph.ai_node import AINode
+from ai_flow.api.ai_flow_context import init_ai_flow_context
+from ai_flow.cli import cli_parser
+from ai_flow.context.workflow_config_loader import current_workflow_config
+from ai_flow.endpoint.server.server import AIFlowServer
+from ai_flow.scheduler_service.service.config import SchedulerServiceConfig
+from ai_flow.test.util.notification_service_utils import start_notification_server, stop_notification_server
+from ai_flow.util.path_util import get_file_dir, get_parent_dir
+
+_SQLITE_DB_FILE = 'aiflow.db'
+_SQLITE_DB_URI = '%s%s' % ('sqlite:///', _SQLITE_DB_FILE)
+_PORT = '50051'
+_SERVER_URI = 'localhost:{}'.format(_PORT)
+_SCHEDULER_CLASS = 'ai_flow.test.api.mock_plugins.MockScheduler'
+_WORKFLOW_NAME = 'test_workflow_operation'
+
+PROJECT_PATH = get_parent_dir(get_parent_dir(get_file_dir(__file__)))
+
+
+class TestCommand(unittest.TestCase):
```

Comment:
Seems no test case in this file.

Reply:
This is the basic test class for the workflow and job command.